### ISSUE TITLE
os-base/scripts/download should be proxy configurable

### ISSUE THREAD

**Issue by bcg62** [rancher/os#475]
I'm attempting to build my own base os and unfortunately I am behind a corporate proxy.

The curl attempts fail with out some hackery to inject http_proxy/https_proxy env settings.

It would be great if it respected your local env settings, or if there was a way to pass the configuration into the container.

Thanks

**Comment by ibuildthecloud** [comment]
Yes we really should respect and propagate the HTTP proxy env variables.  That seems quite doable.

**Comment by SvenDowideit** [comment]
and the same for the certs? https://forums.rancher.com/t/how-to-add-custom-ca-certificate-for-system-docker-use/3924

**Comment by bcg62** [comment]
+1 ... I'm also now in a build env where i have to deal with SSL inspection certs.

**Comment by SvenDowideit** [comment]
unassign me.
